config/initializers/assets.rb

```diff
@@ -11,3 +11,11 @@
 # Add Yarn node_modules folder to the asset load path.
 Rails.application.config.assets.paths << Rails.root.join('node_modules')
 Rails.application.config.assets.precompile << %w[jquery.js jquery_ujs.js]
+
+# Manually add chosen-rails assets paths since the gem's engine doesn't load for Rails 8
+chosen_gem_path = Gem.loaded_specs['chosen-rails']&.full_gem_path
+if chosen_gem_path
```

Comment:
is there any case where this is not present? if this implies that the app will break at a latter point maybe we can just let it break here